paper-tooltip.html

```diff
@@ -106,9 +112,35 @@
         },
 
         /**
+         * Positions the tooltip to the top, right, bottom, left of its content.
+         */
+        position: {
+          type: String,
+          value: 'bottom'
+        },
+
+        /**
+         * If true, no parts of the tooltip will ever be shown offscreen.
```

Comment:
Is `clip` the right word to describe this? Clipping to me implies that part of the thing will be "clipped off". (Sorry, I know this is a huge :bike: :house:)